src/common/transformations/tests/op_conversions/scaled_dot_product_decomposition_test.cpp

```diff
@@ -329,8 +334,70 @@ const std::shared_ptr<ov::Node> scaled_dot_product_attention_decomposition(std::
+        scaled_atten = std::make_shared<ov::op::v8::Softmax>(scaled_attn_sink, -1);
+
+        auto seq_len = std::make_shared<ov::op::v8::Gather>(q_shape, minus_two, zero_i);
+        scaled_atten = std::make_shared<ov::op::v8::Slice>(scaled_atten, zero_i, seq_len, one_i, minus_one);
+    } else {
+        scaled_atten = std::make_shared<ov::op::v8::Softmax>(scaled_atten, -1);
+    }
 
     const std::shared_ptr<ov::Node> result = std::make_shared<ov::op::v0::MatMul>(scaled_atten, value);
     return result;
 }
+
+TEST_F(TransformationTestsF, ScaledDotProductAttentionDecomposition_Sinks) {
+    const PartialShape query_shape{-1, 64, -1, 64};
+    const PartialShape key_shape{-1, 64, -1, 64};
+    const PartialShape value_shape{-1, 64, -1, 64};
+    const PartialShape attention_mask_shape{-1, 1, -1, -1};
+    const PartialShape scale_shape{1};
+    const PartialShape sinks_shape{1, 64, 1, 1};
+
+    auto scale = std::make_shared<ov::op::v0::Parameter>(element::f16, scale_shape);
+    auto query = std::make_shared<ov::op::v0::Parameter>(element::f16, query_shape);
+    auto key = std::make_shared<ov::op::v0::Parameter>(element::f16, key_shape);
+    auto value = std::make_shared<ov::op::v0::Parameter>(element::f16, value_shape);
+    auto attention_mask = std::make_shared<ov::op::v0::Parameter>(element::f16, attention_mask_shape);
+    auto sinks = std::make_shared<ov::op::v0::Parameter>(element::f16, sinks_shape);
+
+    const bool casual = false;
+
+    {
+        auto sdpa = std::make_shared<ov::op::v13::ScaledDotProductAttention>(query,
+                                                                             key,
+                                                                             value,
+                                                                             attention_mask,
+                                                                             scale,
+                                                                             sinks,
+                                                                             casual);
+        model = std::make_shared<ov::Model>(OutputVector{sdpa},
+                                            ParameterVector{query, key, value, attention_mask, scale, sinks});
+        manager.register_pass<ov::pass::VisualizeTree>("before.svg");
+        manager.register_pass<ov::pass::ScaledDotProductAttentionDecomposition>();
+        manager.register_pass<ov::pass::VisualizeTree>("after.svg");
+    }
+
+    {
+        auto ref =
+            scaled_dot_product_attention_decomposition(query, key, value, attention_mask, scale, casual, false, sinks);
+        model_ref = std::make_shared<ov::Model>(OutputVector{ref},
+                                                ParameterVector{query, key, value, attention_mask, scale, sinks});
+        ov::pass::VisualizeTree("model_ref.svg").run_on_model(model_ref);
```

Comment:
Is the VisualizeTree pass here a debug leftover?
It will generate image with transformations unit test run by default (should be rather conditional or removed).